### PR TITLE
[GR-1235] Call ready views use clusters instead of reads

### PR DESCRIPTION
**Pull request by slazicoicr**
I will update Wiki.

Also fixed up Single Lane TS to remove Total Read vestiges as pointed out in https://github.com/oicr-gsi/dashi/pull/273#discussion_r498838582

### CODE REVIEW

File: application/dash_application/utility/sidebar_utils.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import gsiqcetl.column
 from . import df_manipulation as df_tools
 
-total_reads_cutoff_label = "Passed Filter Reads (* 10^6) minimum"
+total_clusters_cutoff_label = "Passed Filter Clusters (* 10^6) minimum"
```
Comment (avarsava) [inline application/dash_application/utility/sidebar_utils.py]: Is this different from clusters_per_sample_cutoff_label at line 19? 'Clusters Per Sample' was the wording requested in the requirements document.

File: application/dash_application/views/call_ready_rna.py [hunk 2]
```diff
@@ -66,7 +66,8 @@ def dataversion():
     "Total Reads (Passed Filter)": "Total reads passed filter",
```
Comment (avarsava) [inline application/dash_application/views/call_ready_rna.py:66]: Is the Total Reads special_col still required?

File: application/dash_application/utility/sidebar_utils.py (hunk 1 above)
Comment (slazicoicr) [reply application/dash_application/utility/sidebar_utils.py]: Fixed.

File: application/dash_application/views/call_ready_rna.py (hunk 2 above)
Comment (slazicoicr) [reply application/dash_application/views/call_ready_rna.py:66]: This way both reads and cluster metrics will be in the table.

**Review by avarsava — APPROVED** [review]

**Review by apmasell — APPROVED** [review]

**Pull request merged**